### PR TITLE
Update federatorai integration

### PR DESCRIPTION
**Pull request by chenyingtz**
### What does this PR do?

Update federatorai integration README.md

### Motivation

Update federatorai integration README.md

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by jtappa — CHANGES_REQUESTED** [review]
Thanks for these updates, just some small feedback for you.

File: federatorai/README.md [hunk 1]
```diff
@@ -244,23 +251,19 @@ Federator.ai does not include any events.
...
-[16]: https://prophetstor.com/wp-content/uploads/documentation/Federator.ai/Latest%20Version/ProphetStor%20Federator.ai%20User%20Guide.pdf
-[17]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/federatorai/images/plus_icon.png
-[18]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/federatorai/images/add_cluster_window.png
+[8]: https://github.com/DataDog/integrations-extras/blob/master/federatorai/metadata.csv
+[9]: https://docs.datadoghq.com/account_management/api-app-keys/
+[10]: https://docs.datadoghq.com/help/
+[11]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/federatorai/images/cluster_overview_dashboard.png
+[12]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/federatorai/images/application_overview_dashboard.png
+[13]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/federatorai/images/cost_analysis_overview.png
+[14]: https://www.youtube.com/watch?v=qX_HF_zZ4BA&t=21s
```
[the remark this answers is not in the document]
Comment (chenyingtz) [reply federatorai/README.md]: hi @jtappa
we have update the wordings, please review.
thanks!

**Comment by chenyingtz** [comment]
hi,
we got this error in PR Changes check.

  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/click/core.py", line 1062, in main
    rv = self.invoke(ctx)
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/click/core.py", line 1668, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/click/core.py", line 763, in invoke
    return __callback(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py", line 21, in validate_profile
    validators.validate_profile(file, directory, verbose)
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/datadog_checks/dev/tooling/commands/meta/snmp/validators/schema_validation.py", line 17, in validate_profile
    profiles_list = find_profiles(file, directory)
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/datadog_checks/dev/tooling/commands/meta/snmp/validators/schema_validation.py", line 87, in find_profiles
    profiles_list = get_all_profiles_from_dir(profiles_path)
  File "/opt/hostedtoolcache/Python/3.8.11/x64/lib/python3.8/site-packages/datadog_checks/dev/tooling/commands/meta/snmp/validators/schema_validation.py", line 94, in get_all_profiles_from_dir
    dir_contents = [file for file in listdir(profiles_path) if isfile(join(profiles_path, file))]
FileNotFoundError: [Errno 2] No such file or directory: '/home/vsts/work/1/s/snmp/datadog_checks/snmp/data/profiles'
##[error]Bash exited with code '1'.
Finishing: Validate snmp profiles

**Comment by hithwen** [comment]
Oh, seems the wrong tests were running here, fixed on https://github.com/DataDog/integrations-core/pull/9710

/azp run

**Review by hithwen — APPROVED** [review]

**Comment by chenyingtz** [comment]
thanks @hithwen!

**Pull request merged**